pkg/mutation/match/match.go

```diff
@@ -22,7 +22,12 @@ const Wildcard = "*"
 // Match selects objects to apply mutations to.
 // +kubebuilder:object:generate=true
 type Match struct {
-	Kinds []Kinds `json:"kinds,omitempty"`
+	// Origin determines if the mutator should be used to expand generator
```

Comment:
This is not just relevant for mutators, but constraints as well (constraints now also use this class)